core/templates/dev/head/forms/formBuilder.js

```diff
@@ -650,7 +672,7 @@ oppia.factory('rteHelperService', [
           elt.find(
             'img.oppia-noninteractive-' + componentDefn.name
           ).replaceWith(function() {
-            var jQueryElt = $('<' + this.className + '/>');
+            var jQueryElt = $('<' + this.className.split(' ')[0] + '/>');
```

Comment:
What's happening here? It might merit a comment.

If you're looking for the class name starting with oppia-noninteractive-*, then taking the first one is error-prone -- you don't know what might happen in the future. Instead I would suggest actually looking for the class name starting with oppia-noninteractive-*, and erroring noisily if you don't find exactly one.